x-pack/plugins/triggers_actions_ui/public/application/sections/rules_list/components/notify_badge/translations.tsx

```diff
@@ -32,3 +32,33 @@ export const OPEN_SNOOZE_PANEL_ARIA_LABEL = i18n.translate(
   'xpack.triggersActionsUI.sections.rulesList.rulesListNotifyBadge.openSnoozePanel',
   { defaultMessage: 'Open snooze panel' }
 );
+
+const SECONDS = i18n.translate('xpack.formatters.timeUnit.seconds', {
```

Comment:
Few nitpicks:

1. Let's use a more component specific translation key: (i.e. `xpack.triggersActionsUI.sections.rulesList.rulesListNotifyBadge.seconds`)

2. We can pluralize these translation strings by doing the following:
```
const getSecondsTranslation = (value: number) => 
  i18n.translate('xpack.triggersActionsUI.sections.rulesList.rulesListNotifyBadge.seconds', {
    defaultMessage: '{value, plural, one {# second} other {# seconds}}',
    values: { value },
  });

getSecondsTranslation(0) -> 0 seconds
getSecondsTranslation(1) -> 1 second
getSecondsTranslation(5) -> 5 seconds
```